Podcast/SeriesDetailHeaderView.swift

```diff
@@ -64,145 +62,219 @@ class SeriesDetailHeaderView: UIView {
         
         subscribeButton = FillButton(type: .subscribe)
         subscribeButton.setTitle("Subscribe", for: .normal)
         subscribeButton.setTitle("Subscribed", for: .selected)
         subscribeButton.addTarget(self, action: #selector(didPressSubscribeButton), for: .touchUpInside)
         
-        settingsButton = UIButton(type: .custom)
-        settingsButton.adjustsImageWhenHighlighted = true
-        settingsButton.setImage(#imageLiteral(resourceName: "settingsButton"), for: .normal)
-        settingsButton.isHidden = true
-        settingsButton.addTarget(self, action: #selector(settingsWasPressed), for: .touchUpInside)
-        
         shareButton = UIButton(type: .custom)
         shareButton.adjustsImageWhenHighlighted = true
         shareButton.setImage(#imageLiteral(resourceName: "shareButton"), for: .normal)
         shareButton.addTarget(self, action: #selector(shareWasPressed), for: .touchUpInside)
         
+        tagsView = UIView()
+        tagsView.backgroundColor = .clear
+        tagsView.clipsToBounds = true
+        
+        viewSeparator = UIView()
+        viewSeparator.backgroundColor = .paleGrey
+
+        infoView.addSubview(backgroundImageView)
+        infoView.addSubview(gradientView)
         infoView.addSubview(imageView)
         infoView.addSubview(titleLabel)
-        infoView.addSubview(publisherLabel)
-        infoView.addSubview(lastEpisodeLabel)
-        infoView.addSubview(lastEpisodeDateLabel)
         infoView.addSubview(subscribeButton)
-        infoView.addSubview(settingsButton)
-        infoView.addSubview(shareButton)
+        infoView.addSubview(tagsView)
+        infoView.addSubview(publisherLabel)
+        infoView.addSubview(viewSeparator)
         
-        viewSeparator = UIView()
-        viewSeparator.backgroundColor = .paleGrey
+        addSubview(infoView)
+        infoView.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.height.equalTo(308.0)
+            make.width.equalToSuperview()
+        }
         
-        tagsView = UIView()
-        tagsView.backgroundColor = .offWhite
-        tagsView.clipsToBounds = true
+//        infoView.addSubview(shareButton)
         
-        relatedTagsLabel = UILabel()
-        relatedTagsLabel.text = "Similar Tags"
-        relatedTagsLabel.textColor = .charcoalGrey
-        relatedTagsLabel.font = ._12RegularFont()
+        backgroundImageView.snp.makeConstraints { make in
+            make.width.equalToSuperview()
+            make.height.equalToSuperview()
+        }
+        
+        gradientView.snp.makeConstraints { make in
+            make.width.equalToSuperview()
+            make.height.equalToSuperview()
+        }
+        
+        imageView.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.top.equalToSuperview().offset(24.0)
+            make.size.equalTo(CGSize(width: imageHeight, height: imageHeight))
```

Comment:
I am pretty sure you can say make.size.equalTo(imageHeight) instead of making a CGSize object